src/main/java/aquality/selenium/core/utilities/ElementActionRetrier.java

```diff
@@ -3,52 +3,25 @@
 import aquality.selenium.core.configurations.IRetryConfiguration;
 import com.google.inject.Inject;
 
-import java.util.Optional;
 import java.util.function.Supplier;
 
-public class ElementActionRetrier implements IElementActionRetrier {
-
-    private IRetryConfiguration retryConfiguration;
+/**
+ * It was renamed to ActionRetrier.
+ */
+public class ElementActionRetrier extends ActionRetrier implements IElementActionRetrier {
```

Comment:
What is the difference between ElementActionRetrier and ActionRetrier?

Reply:
ElementActionRetrier handle only 2 exceptions by default (StaleElementReferenceException.class, InvalidElementStateException.class). 

ActionRetrier should handle different exceptions, depends on parameter.